generator/integration_tests/generator_integration_test.cc

```diff
@@ -93,6 +95,8 @@ class GeneratorIntegrationTest
 
     product_path_ = "generator/integration_tests/golden/";
     googleapis_commit_hash_ = "59f97e6044a1275f83427ab7962a154c00d915b5";
+    copyright_year_ =
+        absl::FormatTime("%Y", absl::Now(), absl::LocalTimeZone());
```

Comment:
I would avoid using a timezone that depends on the environment.  `absl::UTCTimeZone()` is probably most appropriate.